jekyll/_cci2/server/installation/upgrade-server-4.adoc

```diff
@@ -0,0 +1,36 @@
+---
+version:
+- Server v4.x
+- Server Admin
+---
+= Upgrade server v4.x
+:page-layout: classic-docs
+:page-liquid:
+:page-description: "This document lists the steps required to upgrade a CircleCI server v4.x installation."
+:icons: font
+:toc: macro
+:toc-title:
+
+Upgrading your server v4.x installation requires that you have access to the Kubernetes cluster in which server is installed.
+
+Before upgrading, ensure you have set up https://circleci.com/docs/2.0/server-3-operator-backup-and-restore[Backup and Restore]. 
+
+A successful deployment will update the UI. Unless noted in the release notes, updates are rolling updates and there is no downtime. 
+
+NOTE: We recommend that you do not skip releases.
+
+. Before updating ensure there is a recent backup. For more information, see the https://circleci.com/docs/2.0/server-3-operator-backup-and-restore/#creating-backups[Backup and Restore] guide.
+
+. Check the https://circleci.com/server/changelog/[change log] and make sure there are no actions you need to take before deploying a new version. 
+
+. Optionally, confirm what the update is going to do with https://github.com/databus23/helm-diff[Helm Diff]
++
+[source,shell]
+helm diff upgrade server -f <your-yaml.values> <path-to-server-chart>
+
+. Perform the upgrade
++
+[source,shell]
+helm upgrade server -f <your-yaml.values> <path-to-server-chart>
+
+. Deploy and run https://github.com/circleci/realitycheck[`reality check`] in your test environment to ensure your installation is fully operational.
```

Comment:
Hard to tell this is a link. Maybe we can find a way to style cod blocks that are also links, since they end up with no color change.

Reply:
Thanks! I'll add this to the list of design issues 👍